evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -714,3 +717,29 @@ def test_automl_supports_time_series_classification(mock_binary_fit, mock_multi_
 
         assert result['parameters']['Delayed Feature Transformer'] == configuration
         assert result['parameters']['pipeline'] == configuration
+
+
+@pytest.mark.parametrize('problem_type', [ProblemTypes.TIME_SERIES_MULTICLASS, ProblemTypes.TIME_SERIES_BINARY])
```

Comment:
Why are the `TimeSeries` checks in `test_automl_search_classification` and `test_automl_search_regression` but the non-TimeSeries in `test_automl`?

Reply:
Will fix!